### PR TITLE
swiping/liking items feature

### PR DESCRIPTION
**Pull request by JustasBan**
with this feature you can like or/and swipe items from db

### CODE REVIEW

**Comment by Olegas123** [comment]
Maybe could you add a screenshot how it looks like?

File: Controllers/HomeController.cs [hunk 1]
```diff
@@ -33,24 +33,31 @@ public IActionResult Index()
...
+            else
+            {
+                TempData["name"] = "no";
+            }
 
-            return View(itemList.ToList()[current]);
+            return View();
         }
 
         public IActionResult loadNext()
```
Comment (Olegas123) [inline Controllers/HomeController.cs]: loadNext should be LoadNext according to our standartization

**Comment by JustasBan** [comment]
image
everything rendered from databases

File: Controllers/HomeController.cs [hunk 2]
```diff
@@ -6,21 +6,72 @@
...
+            return View(itemList.ToList()[current]);
+        }
+
+        public IActionResult loadNext()
+        {
+            current++;
+            return RedirectToAction("index");
+        }
+
+        public IActionResult likeThis()
```
Comment (Olegas123) [inline Controllers/HomeController.cs]: likeThis should be LikeThis according to our standartization

File: Data/Migrations/20211207224434_addLikedItems.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace kinder_app.Data.Migrations
+{
+    public partial class addLikedItems : Migration
```
Comment (Olegas123) [inline Data/Migrations/20211207224434_addLikedItems.cs:5]: Im not sure, but class name should be from uppercase

Comment (JustasBan) [reply Data/Migrations/20211207224434_addLikedItems.cs:5]: it's a migration class, and i low-key fear that I'll mess up db again. I'll try to fix it day before assignment

**Pull request merged**